WebDavServer.Infrastructure.FileStorage/Services/VirtualStorageService.cs

```diff
@@ -140,7 +140,7 @@ public async Task MoveDirectoryAsync(PathInfo srcPath, PathInfo dstPath, Cancell
             var item = await _dbContext.Set<Item>()
                 .Where(x => x.IsDirectory)
                 .Where(x => x.Title == srcPath.ResourceName)
-                .Where(x => x.DirectoryId == directoryId)
+                .Where(x => x.Id == directoryId)

```

Comment:
in this filter, the item is searched by the parent directory in which it should be located and the name of the items, and not by its id. If we had an id here, it would be enough to do Find(id)

Reply:
From my observations, in sub-directories, that is not in root, this results in not finding any information from the database. Works fine if all directories will always be in root